.github/workflows/testnet.yml

```diff
@@ -2,7 +2,7 @@ name: Testnet Deployment
 
 on:
   push:
-    branches: [ staging-off ]
+    branches: [ staging ]
```

Comment:
this should be branches: [ testnet ] 
or something. There is a real testnet deployment that uses pocket testnet and replicates the production setup.

I don't want this deployment to receive staging code unless we purposefully want it there.